src/apps/dictionaries/logic.ts

```diff
@@ -61,7 +68,7 @@ const recursivelyFetchToConcepts = async (
     );
   }
 
-  return getConceptUrls(mappingsLists);
+  return getConceptUrls(mappingsLists);   
```

Comment:
do we need this change

Reply:
@HerbertYiga yes we need it.

Reply:
> @HerbertYiga yes we need it.

i see the line you added and that you removed are similar!

Reply:
@HerbertYiga it shows she added just space after terminating, it's technically not a change though we should mind about proper formatting. cc @jwnasambu